<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1664542345424' stop='1664542345687'>
    <name>Login procedure</name>
    <title>Login procedure</title>
    <test-cases>
        <test-case start='1664542345424' status='failed' stop='1664542345686'>
            <name>Login Username</name>
            <title>Login Username</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='84abde5f-a820-4694-b2f9-8755eaf99b4b-attachment.png' type='image/png' size='3480'/>
            </attachments>
            <failure>
                <message>Failed: invalid element state: Failed to execute 'replace' on 'Location': 'anurl.com' is not a valid URL.
                  (Session info: chrome=105.0.5195.127)
                  (Driver info: chromedriver=105.0.5195.52 (412c95e518836d8a7d97250d62b29c2ae6a26a85-refs/branch-heads/5195@{#853}),platform=Windows NT 10.0.19044 x86_64)</message>
                <stack-trace>InvalidElementStateError: invalid element state: Failed to execute 'replace' on 'Location': 'anurl.com' is not a valid URL.
                  (Session info: chrome=105.0.5195.127)
                  (Driver info: chromedriver=105.0.5195.52 (412c95e518836d8a7d97250d62b29c2ae6a26a85-refs/branch-heads/5195@{#853}),platform=Windows NT 10.0.19044 x86_64)
                    at Object.checkLegacyResponse (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
                    at parseHttpResponse (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
                    at C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)
                From: Task: Protractor.get(anurl.com) - reset url
                    at Driver.schedule (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
                    at ProtractorBrowser.executeScriptWithDescription (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\built\browser.js:402:28)
                    at C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\built\browser.js:677:25
                    at ManagedPromise.invokeCallback_ (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)
                From: Task: Run it("Login Username") in control flow
                    at UserContext.&lt;anonymous> (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
                    at attempt (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
                    at Spec.execute (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
                    at UserContext.fn (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
                    at attempt (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (D:\Protractorframework\Reuseable\keywords.js:53:5)
                    at addSpecsToSuite (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\RBT\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (D:\Protractorframework\Reuseable\keywords.js:52:1)
                    at Module._compile (node:internal/modules/cjs/loader:1105:14)
                    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
                    at Module.load (node:internal/modules/cjs/loader:981:32)
                    at Function.Module._load (node:internal/modules/cjs/loader:822:12)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>